recipe/meta.yaml

```diff
@@ -24,15 +24,21 @@ requirements:
   run:
     - python
     - mccabe >=0.7.0,<0.8.0
-    - pycodestyle >=2.11.0,<2.12.0
+    - pycodestyle >=2.12.0,<2.13.0
     - pyflakes >=3.2.0,<3.3.0
 
 test:
   imports:
     - flake8
+    - flake8.api
+    - flake8.formatting
+    - flake8.main
+    - flake8.options
+    - flake8.plugins
   requires:
     - pip
   commands:
+    # Cannot run pytest since source does not come with tests.
```

Comment:
likewise with pycodestyle you could use the github url. Not suggesting you do that necessarily but I think the comment is a bit inaccurate/referring to generic packaging concerns and should be removed